src/Compilers/CSharp/Test/Symbol/Symbols/AnonymousTypesSemanticsTests.cs

```diff
@@ -589,6 +1042,32 @@ public static void Test1(int x)
         }
 
         [Fact()]
+        public void AnonymousTypeSymbols_Error_InUsingStatement_OperationTree()
+        {
+            string source = @"
+class ClassA
+{
+    public static void Test1(int x)
+    {
+        using (var v1 = /*<bind>*/new { }/*</bind>*/)
```

Comment:
Sure.

---
In reply to: [123879744](https://github.com/dotnet/roslyn/pull/20364#discussion_r123879744) [](ancestors = 123879744)